lang/README.md

```diff
@@ -1,7 +1,7 @@
 ## Contribute
 
-If you want to improve an existing Matomo translation or contribute a new translation, please have a look at our [Transifex project](https://translations.piwik.org).
+If you want to improve an existing Matomo translation or contribute a new translation, please have a look at our [Weblate project](https://translations.piwik.org).
```

Comment:
```suggestion
If you want to improve an existing Matomo translation or contribute a new translation, please have a look at our [Weblate project](https://hosted.weblate.org/engage/matomo/).
```
Or we could link to https://matomo.org/translations

Reply:
or we use https://translations.matomo.org/, which should redirect as well.

Reply:
Indeed, I added a point to https://github.com/matomo-org/matomo/issues/17907 to update the redirect.